test/game/player/player_component_test.dart

```diff
@@ -63,22 +89,25 @@ void main() {
 
   group('Player', () {
     tester.test('loads correctly', (game) async {
+      await game.ready();
       final player = Player();
-      await game.ensureAdd(player);
+      await game.firstChild<FlameMultiBlocProvider>()?.ensureAdd(player);
```

Comment:
This is an unfortunate side effect of how coupled the `PlayerComponent` is to its game class. Ideally we wouldn't use `VeryGoodAdventuresGame` class to test the component, but since something like `flame_behaviors` wasn't a thing back when we implement this, we ended up with this.

I think we shouldn't address this issue on this PR. But rather leave it as is and this will open the opportunity for us to write a follow up article explaining all the benefits of flame behavior,